src/app.js

```diff
@@ -1,21 +1,21 @@
 'use strict';
 
-/**
- * Implement sum function:
- *
- * Function takes 2 numbers and returns their sum
- *
- * sum(1, 2) === 3
- * sum(1, 11) === 12
- *
- * @param {number} a
- * @param {number} b
- *
- * @return {number}
- */
-function sum(a, b) {
-  // write code here
-  return a + b;
+const http = require('http');
+
+function app() {
```

Comment:
Consider using an arrow function for the 'app' function if you do not need to use 'this' inside the function.